docs/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Leader-board</title>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+<body>
+  <main>
+    <nav>
+      <div id="nav-header">
+        <img src="https://library.sportingnews.com/styles/twitter_card_120x120/s3/2022-01/Beijing-Olympic-medals-013122-Getty-FTR.jpg?itok=j0IAWute'" id="logo" alt="logo">
+        <h1>LEADER-BOARD</h1>
+      </div>
+      <span id="menu">
+        <input type="text" minlength="5" maxlength="15" id="search" placeholder="Search">
+        <a href="https://github.com/ademibowale/Leader-Board" target="_blank">View source</a>
+      </span>
+    </nav>
+    <div class="container">
+      <section id="showcase">
+        <div id="showcase-title">
+          <h2>Recent scores</h2>
+          <h3 id="result"></h3>
+          <button id="refresh">Refresh</button>
```

Comment:
**[OPTIONAL]** The `Refresh` button text is not displayed since its color is white kindly make sure to make it black or other color.

<img width="505" alt="image" src="https://user-images.githubusercontent.com/34578631/180182787-28bbcf79-d9b9-46eb-88c0-b1b29e85100c.png">

Reply:
Thank you